packages/datepicker/src/utils/offset.ts

```diff
@@ -26,3 +27,43 @@ export const getNextOffsetDate = (
   }
   return nextDate;
 };
+
+export const getEdgedOffsetDate = (
+  offsetDate: Date,
+  { days, months, years }: DPOffsetValue,
+  dateEdge?: Date,
+): Date => {
+  if (dateEdge) {
```

Comment:
I would prefer return first approach here
```suggestion
  if (!dateEdge) return offsetDate;
```
It will reduce 1 lvl of ifs

Reply:
done